About.html

```diff
@@ -35,7 +35,7 @@ <h2>About Adegboyega</h2>
                 <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
                 <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
                 <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
```

Comment:
`alt` value should not be empty

Reply:
Thank you for noting it. 1 more thing learned.  I will add alt values and push today.

Reply:
fixed